tests/system_tests_multicast.py

```diff
@@ -429,6 +438,128 @@ def test_999_check_for_leaks(self):
         self._check_for_leaks()
 
 
+class MulticastMeshTest(TestCase):
+    """
+    Verify the multicast forwarding logic across a mesh router
+    configuration
+    """
+    @classmethod
+    def setUpClass(cls):
+        """Start a router"""
+        super(MulticastMeshTest, cls).setUpClass()
+
+        def router(name, mode, extra):
+            config = [
+                ('router', {'mode': mode,
+                            'id': name,
+                            'workerThreads': W_THREADS}),
+                ('listener', {'role': 'normal',
+                              'port': cls.tester.get_port(),
+                              'maxFrameSize': MAX_FRAME,
+                              'linkCapacity': LINK_CAPACITY}),
+                ('address', {'prefix': 'multicast', 'distribution': 'multicast'}),
+            ]
+
+            if extra:
+                config.extend(extra)
+            config = Qdrouterd.Config(config)
+            cls.routers.append(cls.tester.qdrouterd(name, config, wait=True))
+            router_instance = cls.routers[-1]
```

Comment:
This `router_instance` is unused and hence can be removed. The test passed for me without it !

Reply:
Removed `router_instance`.